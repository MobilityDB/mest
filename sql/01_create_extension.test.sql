CREATE EXTENSION IF NOT EXISTS mest CASCADE;

DROP TABLE IF EXISTS tbl_bigint;
CREATE TABLE tbl_bigint(k integer PRIMARY KEY, b bigint);
\copy tbl_bigint FROM 'data/tbl_bigint.data'

DROP TABLE IF EXISTS tbl_int8range;
CREATE TABLE tbl_int8range(k integer PRIMARY KEY, b int8range);
\copy tbl_int8range FROM 'data/tbl_int8range.data'

DROP TABLE IF EXISTS tbl_int8multirange;
CREATE TABLE tbl_int8multirange(k integer PRIMARY KEY, b int8multirange);
\copy tbl_int8multirange FROM 'data/tbl_int8multirange.data'

DROP TABLE IF EXISTS tbl_date;
CREATE TABLE tbl_date(k integer PRIMARY KEY, d date);
\copy tbl_date FROM 'data/tbl_date.data'

DROP TABLE IF EXISTS tbl_daterange;
CREATE TABLE tbl_daterange(k integer PRIMARY KEY, d daterange);
\copy tbl_daterange FROM 'data/tbl_daterange.data'

DROP TABLE IF EXISTS tbl_datemultirange;
CREATE TABLE tbl_datemultirange(k integer PRIMARY KEY, d datemultirange);
\copy tbl_datemultirange FROM 'data/tbl_datemultirange.data'

DROP TABLE IF EXISTS tbl_int;
CREATE TABLE tbl_int(k integer PRIMARY KEY, i integer);
\copy tbl_int FROM 'data/tbl_int.data'

DROP TABLE IF EXISTS tbl_int4range;
CREATE TABLE tbl_int4range(k integer PRIMARY KEY, i int4range);
\copy tbl_int4range FROM 'data/tbl_int4range.data'

DROP TABLE IF EXISTS tbl_int4multirange;
CREATE TABLE tbl_int4multirange(k integer PRIMARY KEY, i int4multirange);
\copy tbl_int4multirange FROM 'data/tbl_int4multirange.data'

DROP TABLE IF EXISTS tbl_path;
CREATE TABLE tbl_path(k integer PRIMARY KEY, p path);
\copy tbl_path FROM 'data/tbl_path.data'

DROP TABLE IF EXISTS tbl_timestamptz;
CREATE TABLE tbl_timestamptz(k integer PRIMARY KEY, t timestamp with time zone);
\copy tbl_timestamptz FROM 'data/tbl_timestamptz.data'

DROP TABLE IF EXISTS tbl_tstzrange;
CREATE TABLE tbl_tstzrange(k integer PRIMARY KEY, t tstzrange);
\copy tbl_tstzrange FROM 'data/tbl_tstzrange.data'

DROP TABLE IF EXISTS tbl_tstzmultirange;
CREATE TABLE tbl_tstzmultirange(k integer PRIMARY KEY, t tstzmultirange);
\copy tbl_tstzmultirange FROM 'data/tbl_tstzmultirange.data'

