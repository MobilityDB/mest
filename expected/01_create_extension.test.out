CREATE EXTENSION IF NOT EXISTS mest CASCADE;
DROP TABLE IF EXISTS tbl_bigint;
NOTICE:  table "tbl_bigint" does not exist, skipping
CREATE TABLE tbl_bigint(k integer PRIMARY KEY, b bigint);
\copy tbl_bigint FROM 'data/tbl_bigint.data'
DROP TABLE IF EXISTS tbl_int8range;
NOTICE:  table "tbl_int8range" does not exist, skipping
CREATE TABLE tbl_int8range(k integer PRIMARY KEY, b int8range);
\copy tbl_int8range FROM 'data/tbl_int8range.data'
DROP TABLE IF EXISTS tbl_int8multirange;
NOTICE:  table "tbl_int8multirange" does not exist, skipping
CREATE TABLE tbl_int8multirange(k integer PRIMARY KEY, b int8multirange);
\copy tbl_int8multirange FROM 'data/tbl_int8multirange.data'
DROP TABLE IF EXISTS tbl_date;
NOTICE:  table "tbl_date" does not exist, skipping
CREATE TABLE tbl_date(k integer PRIMARY KEY, d date);
\copy tbl_date FROM 'data/tbl_date.data'
DROP TABLE IF EXISTS tbl_daterange;
NOTICE:  table "tbl_daterange" does not exist, skipping
CREATE TABLE tbl_daterange(k integer PRIMARY KEY, d daterange);
\copy tbl_daterange FROM 'data/tbl_daterange.data'
DROP TABLE IF EXISTS tbl_datemultirange;
NOTICE:  table "tbl_datemultirange" does not exist, skipping
CREATE TABLE tbl_datemultirange(k integer PRIMARY KEY, d datemultirange);
\copy tbl_datemultirange FROM 'data/tbl_datemultirange.data'
DROP TABLE IF EXISTS tbl_int;
NOTICE:  table "tbl_int" does not exist, skipping
CREATE TABLE tbl_int(k integer PRIMARY KEY, i integer);
\copy tbl_int FROM 'data/tbl_int.data'
DROP TABLE IF EXISTS tbl_int4range;
NOTICE:  table "tbl_int4range" does not exist, skipping
CREATE TABLE tbl_int4range(k integer PRIMARY KEY, i int4range);
\copy tbl_int4range FROM 'data/tbl_int4range.data'
DROP TABLE IF EXISTS tbl_int4multirange;
NOTICE:  table "tbl_int4multirange" does not exist, skipping
CREATE TABLE tbl_int4multirange(k integer PRIMARY KEY, i int4multirange);
\copy tbl_int4multirange FROM 'data/tbl_int4multirange.data'
DROP TABLE IF EXISTS tbl_path;
NOTICE:  table "tbl_path" does not exist, skipping
CREATE TABLE tbl_path(k integer PRIMARY KEY, p path);
\copy tbl_path FROM 'data/tbl_path.data'
DROP TABLE IF EXISTS tbl_timestamptz;
NOTICE:  table "tbl_timestamptz" does not exist, skipping
CREATE TABLE tbl_timestamptz(k integer PRIMARY KEY, t timestamp with time zone);
\copy tbl_timestamptz FROM 'data/tbl_timestamptz.data'
DROP TABLE IF EXISTS tbl_tstzrange;
NOTICE:  table "tbl_tstzrange" does not exist, skipping
CREATE TABLE tbl_tstzrange(k integer PRIMARY KEY, t tstzrange);
\copy tbl_tstzrange FROM 'data/tbl_tstzrange.data'
DROP TABLE IF EXISTS tbl_tstzmultirange;
NOTICE:  table "tbl_tstzmultirange" does not exist, skipping
CREATE TABLE tbl_tstzmultirange(k integer PRIMARY KEY, t tstzmultirange);
\copy tbl_tstzmultirange FROM 'data/tbl_tstzmultirange.data'
